generators/server/templates/src/main/java/package/web/rest/_AccountResource.java

```diff
@@ -231,8 +235,12 @@ public User getAccount(Principal principal) {
                             it.getAuthority()).collect(Collectors.toSet())
                     );
                 }
-            )
-            .orElseThrow(RuntimeException::new);
+            );
```

Comment:
`.orElseThrow(() -> new DefaultException("User could not be found"));` would be better